### ISSUE TITLE
Improve disk size communication

### ISSUE THREAD

**Issue by jniederm** [oVirt/ovirt-web-ui#508]
Current rest api properties and their UI namings
`actual_size` -> Used
`provisioned_size` -> Total
This may be suitable for "thin provisioned" (cow) disks (I'm not sure if size of snapshots is included in actual size). However for "preallocated" (raw) disks it feels pretty misleading since a newly created disk seems to be fully used right from the beginning (see screenshot).

version: commit 852ab59

![image](https://user-images.githubusercontent.com/5278760/36916815-4a2c4f28-1e55-11e8-8af7-af0f0211e98c.png)

**Comment by michalskrivanek** [comment]
how do you suggest to improve it?

**Comment by jniederm** [comment]
Based on IRC conversation with Tal: *actual_size* is allocated size, i.e. how much space does the disk occupy on the underlying storage.

Since the disk may use for example MBR table for partitioning and have not partitions created, there is not notion how much space is actually used by meaningful data. So my suggestions are pairs (`actual_size` / ` provisioned_size`)
* allocated size / provisioned size
* allocated size / (without any adjective) size
* leave only the `provisioned_size` part named as *Size*, since VM Portal users are rarely expected to care about actual allocated size.

Comments? @michalskrivanek, @jelkosz, @mareklibra

**Comment by jelkosz** [comment]
>     allocated size / provisioned size
>     allocated size / (without any adjective) size
>     leave only the provisioned_size part named as Size, since VM Portal users are rarely expected to care about actual allocated size.

^^^ this sounds like the best option to me ^^^

**Comment by mareklibra** [comment]
Rendering just the `provisioned_size` does not help in identification that particaulr VM is running out of disk space.

> Since the disk may use for example MBR table for partitioning and have not partitions created,
This is less common case in my POV

**Comment by jniederm** [comment]
> Rendering just the provisioned_size does not help in identification that particaulr VM is running out of disk space.

Yes, it doesn't. Sadly AFAIK there is no property of [Disk][1] entity that could help in identifying log disk space. As mentioned in the first comment, `actual_size` = `provisioned_size` for all raw disks. And that doesn't mean they are full.

Here is how engine [identifies][2] low disk space.

[1]: http://ovirt.github.io/ovirt-engine-api-model/master/#types/disk
[2]: https://github.com/oVirt/ovirt-engine/blob/f50c2b5a30e762eb10271c4e81712a74c57be04c/backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/monitoring/HostMonitoring.java#L539

**Comment by michalskrivanek** [comment]
actual size is useful, and that's enough.
guest-agent reported values make sense, but only once we have it.

**Comment by gregsheremeta** [comment]
@lizsurette @sjd78 let's discuss this in the next redesign meeting

> guest-agent reported values make sense, but only once we have it

sounds similar to what @michalskrivanek brought up last meeting, where the Disks card was showing Domain-level but needs to show VM-level

**Comment by sjd78** [comment]
This issue is relevant to the disk charts on the Utilization card on #652

@gregsheremeta - Yeah, the discussion was around showing the VM users disk information that is relevant to them.  That means showing them info from the guest (partitions, file system sizes, things that a `df -h` may show), instead of info from infrastructure (provisioned disk size vs allocated disk size in a spare/thin disk/vol).

How exactly that translated to a donut + spark line chart I'm not quite sure.

**Comment by sjd78** [comment]
See PR #700

**Comment by gregsheremeta** [comment]
no longer relevant
